src/Tests/Unit/DebugBranches.test.ts

```
@@ -0,0 +1,57 @@
+    expect(spy).toHaveBeenCalledTimes(1);
+    expect(result).toBe('spy-result');
+  });
+});
+
+describe('getDebug — child logger module name', () => {
+  const logLevels = ['trace', 'debug', 'info', 'warn', 'error', 'fatal'] as const;
+
+  it.each(logLevels)('logs at %s level without error', level => {
+    const log = getDebug('branch-test');
+    expect(() => {
+      (log[level] as (...a: unknown[]) => string)('test');
```

ESLint

[error] 'a' is defined but never used.

(no-unused-vars)